hrc_speech_prediction/tests/test_context_model.py

```diff
@@ -0,0 +1,86 @@
+from unittest import TestCase
+
+from hrc_speech_prediction.data import ALL_ACTIONS
+from hrc_speech_prediction.context_model import ContextTreeModel, Node
+
+
+class TestNode(TestCase):
+
+    def setUp(self):
```

Comment:
I know that this was the case also in the previous version, but for uniformity I would change the name of this function from `setUp` to `set_up`